packages/agoric-cli/src/cosmos.js

```diff
@@ -50,7 +50,8 @@ export default async function cosmosMain(progname, rawArgs, powers, opts) {
                 },
               );
               // Ensure the build doesn't mess up stdout.
-              ps.childProcess.stdout.pipe(process.stderr);
+              // @ts-expect-error stdout won't be null
+              ps.childProcess?.stdout.pipe(process.stderr);
```

Comment:
Was this a mistaken optional chaining of childProcess?  Maybe you meant:
```suggestion
              ps.childProcess.stdout?.pipe(process.stderr);
```